ipa-core/src/test_fixture/world.rs

```diff
@@ -1329,4 +1317,45 @@ mod tests {
             assert_eq!(first_result, second_result);
         });
     }
+
+    #[test]
+    fn cross_shard_prss() {
```

Comment:
is it worth also adding a test to ensure that regular PRSS is *not* consistent across shards?

Reply:
Great suggestion, we do already have it. It was implemented with in-memory sharding. Check `unique_prss_per_shard` test in this file